hexrd/wppf/phase.py

```diff
@@ -298,6 +342,31 @@ def CalcStar(self, v, space, applyLaue=False):
                 vsym = np.vstack((vsym, vp))
         return vsym
 
+    def removeinversion(self, ksym):
+        """
+        this function chooses a subset from a list 
+        of symmetrically equivalent reflections such
+        that there are no g and -g present.
+        """
+        klist = []
+        for i in range(ksym.shape[0]):
+            k = ksym[i,:]
+            kk = list(k)
+            nkk = list(-k)
+            if klist == []:
```

Comment:
```suggestion
            if not klist:
```

This is the standard way to check for an empty list.